includes/classes/legacy/class-tour-operator.php

```
@@ -38,6 +38,30 @@ class Tour_Operator {
 	 */
 	public $framework = false;
 
+	/**
+	 * Holds the admin class instance.
+	 *
+	 * @since 1.0.0
+	 * @var object
+	 */
+	public $admin = false;
```

PHPStan (2.1.46)

[warning] Property lsx\legacy\Tour_Operator::$admin (object) does not accept default value of type false.


(property.defaultValue)